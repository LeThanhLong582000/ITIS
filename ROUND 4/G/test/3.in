4
59
26 21 30 59 55 51 38 47 34 29 47 43 58 22 10 45 44 15 58 29 18 24 2 22 5 7 18 43 16 53 16 10 8 16 29 39 49 4 4 23 41 37 39 17 11 2 19 52 11 16 55 36 2 45 45 25 18 16 49 
38
11 28 5 34 10 9 10 7 4 37 5 20 30 9 10 12 17 10 16 8 13 29 28 30 11 37 1 23 1 17 5 38 6 25 24 11 38 14 
84
18 75 81 52 70 9 5 83 36 70 74 51 84 5 59 61 33 24 79 55 58 17 26 20 48 20 29 51 51 1 31 25 9 35 31 76 15 34 27 44 15 29 62 37 82 50 53 35 60 22 56 76 4 68 45 65 28 84 40 51 23 31 48 33 39 21 47 57 7 37 4 48 47 43 11 22 73 17 25 17 3 38 79 79 
56
13 25 42 13 55 30 50 52 33 14 51 18 39 3 7 50 13 3 35 36 6 18 15 16 8 18 49 8 37 24 41 37 56 3 2 9 47 34 16 20 25 6 49 31 13 55 28 31 4 37 37 8 39 23 3 32 
