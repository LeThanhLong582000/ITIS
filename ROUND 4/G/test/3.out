29
16
40
27
